packages/perseus/src/widgets/radio/multiple-choice-component.new.tsx

```diff
@@ -72,208 +37,102 @@ interface MultipleChoiceComponentProps {
  * Created as part of the Radio Revitalization Project (LEMS-2933).
  */
 const MultipleChoiceComponent = ({
-    apiOptions,
-    reviewModeRubric,
-    reviewMode,
-    editMode = false,
-    multipleSelect = false,
-    labelWrap,
+    choices,
     countChoices,
+    multipleSelect = false,
     numCorrect,
-    isLastUsedWidget,
-    choices,
     onChoiceChange,
+    reviewMode,
 }: MultipleChoiceComponentProps): React.ReactElement => {
     const {strings} = usePerseusI18n();
 
-    const choiceRefs = useRef<Array<React.RefObject<HTMLButtonElement>>>([]);
-    // Keep track of the previous review mode rubric to avoid unnecessary
-    // scrolling when switching between review mode and edit mode.
-    const prevReviewModeRubric = useRef<
-        PerseusRadioWidgetOptions | undefined | null
-    >();
-
-    useEffect(() => {
```

Comment:
I'm so glad we were able to get rid of this useEffect, as it just took up a huge portion of this file